docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Frontend Shared Package
```

Comment:
I'd suggest something like "Development guide" rather than repeating the package name. Also the sentence below should say something like "This guide explains how to make changes to the frontend-shared and use them with your local copy of the client", rather than explaining what the package is.